packages/core/package.json

```
@@ -3,7 +3,7 @@
   "version": "3.0.1",
   "main": "dist/index.cjs",
   "module": "dist/index.mjs",
-  "types": "dist/index.d.ts",
+  "types": "dist/types.d.ts",
```

```suggestion
  "types": "dist/index.d.ts",
```

`index.d.ts` exports all type declarations (including functions and objects), `typed.d.ts` is just exporting common interface declarations and type aliases.